webpack/JobWizard/steps/HostsAndInputs/SelectedChips.js

```diff
@@ -31,7 +33,7 @@ const SelectedChip = ({ selected, setSelected, categoryName }) => {
             onClick={() => deleteItem(id)}
             closeBtnAriaLabel={`Remove ${name}`}
           >
-            {name}
+            {displayFqdnForHosts ? name : name.split('.')[0]}
```

Comment:
`selected` variable is a set of hosts returned by the API, they all have `display_name` property, you can use it here, as long as you change the `map` statement:
instead of:
``` js
selected.map(({ name, id }, index) => (...)
```
You can add the `display_name`:
``` js
selected.map(({ name, display_name, id }, index) => (...)
```

Reply:
I think we are using here gql for the query, which means that we don't have access to the `display_name` property.

Reply:
Looks like we are missing `display_name` from GQL definition in foreman. Let's add it first.

Reply:
https://github.com/theforeman/foreman/pull/9968 added to the description